src/styles.scss

```diff
@@ -264,3 +264,9 @@ mat-icon {
 .mat-tooltip {
   font-size: 15px !important;
 }
+
+.cdk-overlay-pane {
```

Comment:
what does cdk stand for?

Reply:
.cdk-overlay-pane is material-ui 's class